challenge_3/go/erocs/majority.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+)
+
+func FindMajority(int_array []int) int {
```

Comment:
Go idioms is to use smaller naming conventions. This is also a slice, not an array. Maybe `ints`